src/components/App/App.component.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-/* eslint-disable */
+/* eslint-disable import/first */
```

Comment:
The reason I didn't: this would change the insertion order of the css into the DOM. Right now the css from app.css is inserted before everything else in the file and the files dependencies. It think it makes sense to insert the app.css file first in case you want to override something.